README.md

```diff
@@ -1,280 +1,53 @@
 ﻿# ARCtrl
-Top level ARC DataModel and API function descriptions.
 
-![Nuget](https://img.shields.io/nuget/dt/ARCtrl?label=nuget&link=https%3A%2F%2Fwww.nuget.org%2Fpackages%2FARCtrl)
-![npm](https://img.shields.io/npm/dt/%40nfdi4plants%2Farctrl?label=npm&link=https%3A%2F%2Fwww.npmjs.com%2Fpackage%2F%40nfdi4plants%2Farctrl)
+> **ARCtrl** the easy way to read, manipulate and write ARCs in __.NET__ 
+and __JavaScript__! ❤️
 
-- [ARCtrl](#arctrl)
-  - [Jargon/Nomenclature](#jargonnomenclature)
-  - [API Design](#api-design)
-  - [Top level overview](#top-level-overview)
-  - [Stack](#stack)
-- [Libraries](#libraries)
-  - [Design choices](#design-choices)
-    - [Fable compatibility as top priority](#fable-compatibility-as-top-priority)
+| Version | Downloads |
```

Comment:
Maybe we could include a "Tests/Codecov" badge?

Reply:
Yeah thats a bit of work i am not willing to do for this PR. For more information see here:

- https://github.com/TheAngryByrd/MiniScaffold/blob/755f63a4b8daf70f10266b1e124d7c39fffcaac7/Content/Library/build/build.fs#L443
- https://github.com/SteveGilham/altcover